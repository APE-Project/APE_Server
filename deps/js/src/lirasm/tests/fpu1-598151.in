	ptr = allocp 8

	a = immd 1.0
	b = immd 2.5
	c = addd a b
	std c ptr 0
	j done

foo:	std c ptr 0

done:	res = ldd ptr 0
	retd res
