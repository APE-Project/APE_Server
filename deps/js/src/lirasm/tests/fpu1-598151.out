Output is: 3.5
